static/templates/settings/edit_bot_form.hbs

```diff
@@ -31,9 +31,14 @@
         </div>
         <div class="input-group edit-avatar-section">
             <label>Avatar</label>
+            {{!-- Shows the current avatar --}}
+            <img src="{{bot_avatar_url}}"  id="bot_avatar_img" />
```

Comment:
I renamed this to `current_bot_avatar_image`, just to more clearly distinguish it from the editing UI part.